galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -250,9 +250,9 @@ def configure_logging(settings: Dynaconf) -> Dict[str, Any]:
         )
     }
     if data["GALAXY_ENABLE_API_ACCESS_LOG"]:
-        data["INSTALLED_APPS"] = ["automated_logging", "dynaconf_merge"]
+        data["INSTALLED_APPS"] = ["dynaconf_merge"]
         data["MIDDLEWARE"] = [
-            "automated_logging.middleware.AutomatedLoggingMiddleware",
+            # "automated_logging.middleware.AutomatedLoggingMiddleware",
```

Comment:
do we have a plan to replace this? I'm not sure that we can just drop logging as a feature.

Reply:
https://issues.redhat.com/browse/AAH-2388 There is nothing testing for this library's features nor would anyone give me context about what it was being used for.

Reply:
Hoping to solve this in a near future PR.